network/android/src/main/java/com/capacitorjs/plugins/network/NetworkPlugin.java

```diff
@@ -42,7 +42,7 @@ protected void handleOnDestroy() {
     @SuppressWarnings("MissingPermission")
     @PluginMethod
     public void getStatus(PluginCall call) {
-        if (hasRequiredPermissions()) {
+        if (isPermissionDeclared("Network")) {
```

Comment:
there is no `Network` alias, that's the plugin name, I get a 
> isPermissionDeclared: No alias defined for Network or missing @CapacitorPlugin annotation.

Reply:
Thanks. I dont think this plugin needs a permission annotation - the permission isn't dangerous and doesn't require permission checks from the user. We put this permissions in the manifest of the plugin now so it should always be there.

Reply:
yeah, that was what I commented on the first implementation PR, but @ikeith said it did no harm to keep the check

https://github.com/ionic-team/capacitor-plugins/pull/8/files#r477256795